submissions/opeyemiPeter/challenge1-arrays/question2.js

```diff
@@ -0,0 +1,27 @@
+// Question 2
+
+function solution2(nums, target) {
```

Comment:
Nice idea. Well done!
The time complexity is O(n^2) as you rightly said. Can you see if there's any way you could reduce that to O(n)

Think about using [dictionaries in Javascript.](https://www.freecodecamp.org/news/javascript-hash-table-associative-array-hashing-in-js/).

Reply:
yes sure i can, should i and push back?

Reply:
yeah, of course. But you would need to create a new pull request.